### PR TITLE
{chem}[intel/2018a] FLEUR v0.27-MaXR3

### PR DESCRIPTION
**Pull request by migueldiascosta**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
cermis - Linux debian 9.4, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/7ef577644341722c1037efe577fd4c21 for a full test report.

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
cermis - Linux debian 9.4, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/a079f94cbbeb70d43ef794be539fa9c4 for a full test report.

File: easybuild/easyconfigs/f/FLEUR/FLEUR-MaXR2.1-intel-2018a.eb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+easyblock = 'CMakeMake'
+
+name = 'FLEUR'
+version = 'MaXR2.1'
```
Comment (vanzod) [inline easybuild/easyconfigs/f/FLEUR/FLEUR-MaXR2.1-intel-2018a.eb]: Why not using a version like `0.27-MaXR2.1`?

Comment (migueldiascosta) [reply easybuild/easyconfigs/f/FLEUR/FLEUR-MaXR2.1-intel-2018a.eb]: well, I guess that depends on whether there will ever be a `0.28` version :)

but you're certainly right about the continuity aspect, will do

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.9, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/0aed09ba7ae0c29f5a97cc3519d197b6 for a full test report.

File: easybuild/easyconfigs/f/FLEUR/FLEUR-0.27-MaXR3-intel-2018a.eb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+skipsteps = ['install']
+separate_build_dir = True
+
+preconfigopts = 'export FLEUR_LIBRARIES="$EBROOTWANNIER90/lib/libwannier.a;'
+preconfigopts += '-L$EBROOTHDF5/lib;-lhdf5;-lhdf5_fortran;'
+preconfigopts += '-lmkl_scalapack_lp64;-lmkl_blacs_intelmpi_lp64" && '
+
+postinstallcmds = [
+    "mkdir -p %(installdir)s/bin",
+    "cp fleur fleur_MPI inpgen %(installdir)s/bin",
```
Comment (boegel) [inline easybuild/easyconfigs/f/FLEUR/FLEUR-0.27-MaXR3-intel-2018a.eb]: @migueldiascosta please use `cp -a` to preserve permissions & co?

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2109.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/ca0084ac82974227423ba6b49d289b50 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
node3161.skitty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/6c04fd65ba876cac44ebc2bb23ce7f20 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2458.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/01ea8d7dc4236531e45506b7010038cc for a full test report.

**Comment by boegel** [comment]
@migueldiascosta Compilation fails on Skylake, compiler segfaults...

I'm able to dance around that problem using `'lowopt': True` in `toolchainopts`, worth considering?

**Comment by migueldiascosta** [comment]
@boelgel I suppose so - but an internal compiler error is a serious bug, can you try with `intel/2018b` to check if it has already been fixed?

**Comment by boegel** [comment]
@migueldiascosta works fine with `intel/2018b` (without enabling `lowopt`)

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.9, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/fe05aff5275da596908923ef415829b2 for a full test report.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.9, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/b93db163390b86f16a43d877c1994ee0 for a full test report.

**Comment by boegel** [comment]
(close/re-open to let Travis chew on it again with updated tests...)

@migueldiascosta Is this actually still `WIP` as the PR title suggests?

**Comment by migueldiascosta** [comment]
@boegel no, that was before changing the toolchain, fixing that. Will also test again, just in case.

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
grc-cluster1 - Linux centos 6.10, Intel(R) Xeon(R) CPU E5-2640 0 @ 2.50GHz, Python 2.7.14
See https://gist.github.com/11ea46cf00fe5cc26f178471d782311f for a full test report.

**Comment by boegel** [comment]
@migueldiascosta Please mention why you closed this?

**Pull request closed without merging**